packages/cardpay-cli/scheduled-payment/schedule-payment.ts

```diff
@@ -0,0 +1,51 @@
+import { Argv } from 'yargs';
+import { getConstant, getSDK } from '@cardstack/cardpay-sdk';
+import { getEthereumClients, getConnectionType, NETWORK_OPTION_ANY } from '../utils';
+import { Arguments } from 'yargs';
+
+export default {
+  command: 'schedule-payment <senderSafeAddress> <safeModuleAddress> <tokenAddress> <spHash>',
```

Comment:
perhaps this command could actually generate the hash, that way it could be called with more intuitive params

Reply:
I was thinking there could be another command - for generating the spHash. That command could call the `createSpHash` on the contract so that we can have a single source of truth for generating the hash. So since I'm a bit reluctant to putting multiple contract calls in a single CLI command I'm leaning towards having two separate commands - one for generating the shHash and one for using that to schedule a payment.